internal/pkg/log/cli_test.go

```diff
@@ -37,10 +38,15 @@ func TestCliLogger_File(t *testing.T) {
 	assert.NoError(t, file.File().Close())
 
 	// Assert, all levels logged with the level prefix
-	expected := "DEBUG\tDebug msg\nINFO\tInfo msg\nWARN\tWarn msg\nERROR\tError msg\n"
+	expected := ""
+	expected += `{"level":"debug","time":"%s","message":"Debug msg"}` + "\n"
+	expected += `{"level":"info","time":"%s","message":"Info msg"}` + "\n"
+	expected += `{"level":"warn","time":"%s","message":"Warn msg"}` + "\n"
+	expected += `{"level":"error","time":"%s","message":"Error msg"}` + "\n"
+
```

Comment:
https://github.com/keboola/keboola-as-code/blob/5cfcf3c6e908b2c8910f77a501229145f43c5517/internal/pkg/service/common/etcdop/session_test.go#L54-L64

Could we keep this way of writing multi-line texts in tests ^^^? Or do you think it's a bad way?

The `wildcard` package automatically ignores leading and traling `\n`.

With the `assert.Equal` you can use `strings.TrimSpace`.

----------------

My point of view:

Pros:
- It is easy for editing.
- No escaping is needed.
- It provides easy refactoring with regexp, your format would complicate it (we have a lot of such assertions in the code).

Cons:
- It is at the beginning of the line, without indentation.

Reply:
I don't mind using this though I do wish Go had something like PHP's newdoc syntax which can be indented.

It's changed accordingly now.